CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v10.2.0
```

Comment:
Need to bump the version in `package.json` too

Reply:
Oops, good point. I always forget to do it and end up realising when I run `npm publish` 🥇